t30.php

```diff
@@ -79,6 +84,7 @@ public function __construct() {
             ['name' => 'address_supplement', 'type' => 'varchar', 'length' => 255],
             ['name' => 'zip', 'type' => 'varchar', 'length' => 5],
             ['name' => 'city', 'type' => 'varchar', 'length' => 255],
+            ['name' => 'district', 'type' => 'varchar', 'length' => 255],
```

Comment:
Wird der District hier in der Tabelle gespeichert?

Reply:
Das Feld ist der Container, in dem der district dann gelegt wird. Der kann leer gelassen werden.

Reply:
Das finde ich nicht gut. Ich habe den Branch mal installiert und dann ein:

```
show columns from institution;
```
Da taucht dann district auf.

Das verwirrt und verletzt das "Single Source Prinzip"

Reply:
Wenn du es nur so hin bekommst, dann würde ich lieber einen eigenen API Call außerhalb deiner Api machen, als da eine Spalte einzufügen.